application/src/js/cms/_scroll_past_sticky_status_banner.js

```diff
@@ -0,0 +1,30 @@
+;(
+    function () {
+        var errorBox = document.querySelector('.flash-message .alert-box.error');
+
+        if (!isUndefinedOrNull(errorBox)) {
+            var errors = errorBox.querySelectorAll('a')
+
+            var scrollForStickyHeader = function (event) {
+                /* We use a timeout here so that this code runs after the default click event, which adds an anchor
+                 * to the URL and scrolls to the right place. */
+                window.setTimeout(function () {
+                    var href = this.getAttribute('href');
+                    var targetElement = document.querySelector(href);
```

Comment:
This works, but only by the coincidence (?) that the `#blah` fragment reference uses the same syntax as the ID selector in CSS.

It might be a little bit more robust (if more work) to parse out the ID from the href using a regex (`/\#(.+)\z/`?) and then using `document.getElementById()`? That would then also work if you used the full page url followed by the fragment (rather than just the fragment).  However that might be overkill.

Reply:
Seems like probably overkill. But I don't mind doing it. I think technically you're right.

Reply:
The only risk is that we use a longer (but still valid) anchor link at some point, eg `<a href="/current_page#blah">`. Seems like quite a small risk though.

Reply:
Let's accept that risk for now.

Reply:
👍